src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -55,6 +55,17 @@ public static function provideBytesAt(): array
         ];
     }
 
+    public function testWordwrap()
+    {
+        $instance = static::createFromString('Lorem Ipsum');
+        $actual = $instance->wordWrap(5, '-');
```

Comment:
wordwrap

Reply:
OK i will add dataProvider and more assertion here

Reply:
@Nicals  i've added more test cases